xcode/Subconscious/Shared/Library/CollectionUtilities.swift

```diff
@@ -10,7 +10,10 @@ import Foundation
 extension Collection {
     /// A safe bounds-checked array index lookup
     func get(_ index: Index) -> Element? {
-        indices.contains(index) ? self[index] : nil
+        guard index >= startIndex && index < endIndex else {
```

Comment:
While fixing test code using this bounds-checked arraay getter, I realized we could be doing bounds checking more efficiently.

`indices.contains` does a loop over the array (ostensibly). but since indexes are comparable, all we have to do is check if index is between start (inclusive) and end (exclusive)